hera_pspec/pspecdata.py

```diff
@@ -2551,7 +2645,9 @@ def pspec(self, bls1, bls2, dsets, pols, n_dlys=None,
                     # Generate the covariance matrix if error bars provided
                     if store_cov:
                         if verbose: print(" Building q_hat covariance...")
-                        cov_qv = self.cov_q_hat(key1, key2)
+                        #cov_qv = self.cov_q_hat(key1, key2)
```

Comment:
I think this should remain the same since spw_Ndlys is not set by nfreq. The number of total channels that is greater then 0 (including extension channels which Y gives) still needs to exceed the number of spw_Ndlys.

Reply:
I've made a change to compare spw_Nfreq to spw_Ndlys instead.